client/lib/presales-chat/index.ts

```diff
@@ -34,14 +34,21 @@ function getGroupName( keyType: KeyType ) {
 	}
 }
 
-export function usePresalesChat( keyType: KeyType, enabled = true ) {
+export function usePresalesChat( keyType: KeyType, enabled = true, skipAvailabilityCheck = false ) {
 	const isEnglishLocale = useIsEnglishLocale();
 	const isEligibleForPresalesChat = enabled && isEnglishLocale;
 
 	const group = getGroupName( keyType );
-	const { data: chatAvailability, isInitialLoading: isLoadingAvailability } =
-		useMessagingAvailability( group, isEligibleForPresalesChat );
-	const isPresalesChatAvailable = Boolean( chatAvailability?.is_available );
+	const { data: chatAvailability, isInitialLoading } = useMessagingAvailability(
+		group,
+		isEligibleForPresalesChat
+	);
+
+	//some types of chat should always show the widget, even if the chat is not available
+	const isPresalesChatAvailable = skipAvailabilityCheck
+		? true
+		: Boolean( chatAvailability?.is_available );
+	const isLoadingAvailability = skipAvailabilityCheck ? false : isInitialLoading;
```

Comment:
```suggestion
	const { data: chatAvailability, isInitialLoading } = useMessagingAvailability(
		group,
		isEligibleForPresalesChat && ! skipAvailabilityCheck
	);
	
	const isPresalesChatAvailable = skipAvailabilityCheck || Boolean( chatAvailability?.is_available );
```

Some suggestions on how to simplify the code. Most importantly, we should use `skipAvailabilityCheck` to enable or disable the `useMessagingAvailability` hook. Otherwise, we'd be making an unnecessary API call. This in turn means we don't have to have the `const isLoadingAvailability = skipAvailabilityCheck ? false : isInitialLoading;` line, since the hook is disabled, so `isInitialLoading` will be `false`.

Reply:
@klimeryk Great point.  I've implemented your suggestion.  One thing threw me off, we ended up with isIntitialLoading unused and the linter howling about it, so thanks to a tip from @mattgawarecki I destructured isInitialLoading so there would always be some assignment.